src/components/SUE/report/SueReportSend.tsx

```diff
@@ -153,11 +153,14 @@ export const SueReportSend = ({
                   </View>
 
                   <View style={styles.headerContainer}>
+                    <RegularText small style={styles.ratingContainer}>
```

Comment:
`ratingContainer`? can you find a name, that fits both? something like `labelContainer` for example?

Reply:
I named it `labelText`. because this is actually the style of the text component.

Reply:
true! well done!